samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/Models/OktaVerifySelectEnrollmentChannelModel.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Okta.Idx.Sdk;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+
+namespace embedded_auth_with_sdk.Models
+{
+    public class OktaVerifySelectEnrollmentChannelModel
+    {
+        public OktaVerifySelectEnrollmentChannelModel()
+        { 
+        }
+
+        public OktaVerifySelectEnrollmentChannelModel(IAuthenticator authenticator)
+        {
+            this.AuthenticatorId = authenticator.Id;
+        }
+
+        [Required]
+        [Display(Name = "enrollment channel")]
+        public string SelectedChannel
+        {
+            get; 
+            set;
+        }
+
+        [Required]
+        public string AuthenticatorId { get; set; }
+
+        public IList<OktaVerifySelectEnrollmentChannelOptionModel> EnrollmentChannelOptions 
+        {
+            get => new List<string>(new string[] {"email", "sms"})
```

Comment:
It's fine if we hardcode these values in the sample. However a developer should be able to get these values from the authentication response, something like this:

```
authResponse.CurrentAuthenticator.ChannelTypes or use the existing authResponse.CurrentAuthenticator.MethodTypes
```
Are we mapping those values from the response to the authenticator object?

Reply:
The values are not included as part of the current authenticator; only the `select-enrollment-channel` remediation option contains the valid values.  With the current design of the idxclient we would have to make another `introspect` call to retrieve them.

Reply:
A tech debt item if we feel strongly enough about providing these values dynamically.